packaging/python_wheel/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM quay.io/pypa/manylinux1_x86_64
```

Comment:
sigh, this is the way I had it first, but switched over to having only to pull quay.io/pypa/manylinux1_x86_64

Probably better to do it this way.  I would be nice if we could pull this dockerfile out, and use it for all the builds.  I guess that's for the future.